docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
I'm not sure we should name this after the use case. I know we didn't like the term "foreign layers", but hopefully we can come up with something more descriptive:

```
complete-registries
allow-nondistributable-registries
internal-registries
```

Some ideas.